cmd/orch-tester/orch_tester.go

```diff
@@ -71,30 +74,26 @@ func main() {
 		log.Fatal("region is required")
 	}
 
-	metricsURL, err := defaultAddr(*metrics, "127.0.0.1", prometheusPort)
-	if err != nil {
-		log.Fatal(err)
-	}
-
-	streamTesterURL, err := defaultAddr(*streamTester, "127.0.0.1", streamTesterPort)
-	if err != nil {
-		log.Fatal(err)
-	}
-
-	leaderboardURL, err := defaultAddr(*leaderboard, "127.0.0.1", "3001")
-	if err != nil {
-		log.Fatal(err)
-	}
+	ctx, cancel := context.WithCancel(context.Background())
+	defer cancel()
 
-	subgraphURL, err := defaultAddr(*subgraph, "127.0.0.1", "8080")
-	if err != nil {
-		log.Fatal(err)
+	if *streamTester == "" {
+		// start streamtester service
+		glog.Info("Using embedded broadcaster")
+		hostName, _ := os.Hostname()
+		streamtesterMetrics.InitCensus(hostName, model.Version, "streamtester")
+		s := server.NewStreamerServer(false, "", "", 4242)
```

Comment:
Might want to provide `*mistPort` customizability through a CLI flag as well. If you believe it's unnecessary, WDYT of creating a constant for it just so it's clear what it means here? I initially looked at this and was confused thinking this was the port that the stream server would listen too. Especially confusing since there's another port below.

Same for `*mistCreds` and Livepeer `*apiToken`. Or are all of those unused in the use that we make of the strema-tester from the orch-test? (then it's a `StreamerServer` interface issue, no need to bother refactoring).

Reply:
I think `mistCreds` and `apiToken` are not needed (at least I don't see why we could need them. 

Totally, agree with constants, updated in 254c84ad48709ef8f3f622d6ae0cc66b4dd1c702